Cargo.toml

```diff
@@ -24,3 +24,13 @@ bencher = "0.1.5"
 [[bench]]
 name = "bench"
 harness = false
+
+[profile.release]
+codegen-units = 1  # if > 1 enables parallel code generation which improves
+                   # compile times, but prevents some optimizations.
+                   # Passes `-C codegen-units`. Ignored when `lto = true`.
+[profile.test]
+codegen-units = 1
```

Comment:
Does the test profile need to be optimized as much?

Reply:
Good question. 
Thinking again, I think we don't want to apply the change for test build, because we will build for test very often and also want those build quick.

Reply:
done!